doc/source/ray-air/examples/tf_tabular_batch_prediction.py

```diff
@@ -0,0 +1,39 @@
+import numpy as np
+from tensorflow import keras
+from tensorflow.keras import layers
+
+import ray
+from ray.data.preprocessors import Concatenator
+from ray.train.tensorflow import to_air_checkpoint, TensorflowPredictor
+from ray.train.batch_predictor import BatchPredictor
+
+
+def create_model(input_features):
+    return keras.Sequential(
+        [
+            keras.Input(shape=(input_features,)),
+            layers.Dense(16, activation="relu"),
+            layers.Dense(16, activation="relu"),
+            layers.Dense(1, activation="sigmoid"),
+        ]
+    )
+
+
+dataset = ray.data.read_csv("s3://anonymous@air-example-data/breast_cancer.csv")
+num_features = len(dataset.schema().names) - 1
+dataset = dataset.drop_columns(["target"])
```

Comment:
ditto

Reply:
Same here- can you use feature_columns instead of dropping?

Reply:
done